native/fastlane/Fastfile

```diff
@@ -286,6 +286,7 @@ platform :ios do
         force: true,
         skip_screenshots: skip_screenshots,
         skip_metadata: false,
+        overwrite_screenshots: true, # TODO Remove again after next delivery
```

Comment:
We can also just keep this flag set to true

Reply:
You are probably right, would be the more intuitive behavior. Removed the TODO.